src/components/DropZone/README.md

```diff
@@ -142,69 +139,63 @@ Use to pair with a label for better accessibility.
 Use for cases that accept image file formats.
 
 ```jsx
-class DropZoneExample extends React.Component {
-  state = {
-    files: [],
-    rejectedFiles: [],
-    hasError: false,
-  };
-
-  render() {
-    const {files, hasError, rejectedFiles} = this.state;
-
-    const fileUpload = !files.length && <DropZone.FileUpload />;
-    const uploadedFiles = files.length > 0 && (
-      <Stack vertical>
-        {files.map((file, index) => (
-          <Stack alignment="center" key={index}>
-            <Thumbnail
-              size="small"
-              alt={file.name}
-              source={window.URL.createObjectURL(file)}
-            />
-            <div>
-              {file.name} <Caption>{file.size} bytes</Caption>
-            </div>
-          </Stack>
+function DropZoneWithImageFileUpload() {
+  const [files, setFiles] = useState([]);
+  const [rejectedFiles, setRejectedFiles] = useState([]);
+  const [hasError, setHasError] = useState(false);
```

Comment:
Do we need a state to track this? It seems to be derived from other state info so I think we'd be able to do  `const hasError = rejectedFiles.length > 0`